buildfix.sh

```diff
@@ -66,7 +66,7 @@ fi
 
 if ((GAZELLE)); then
   echo "Fixing BUILD deps with gazelle..."
-  CLI_VERSION="5.0.21" # keep in sync with checkstyle.yaml
+  CLI_VERSION="5.0.25" # Update this to latest version from `git lag -l | grep cli-`
```

Comment:
```suggestion
  CLI_VERSION="5.0.25" # Update this to latest version from `git tag -l 'cli-*' --sort=committerdate | tail -n1`
```

Reply:
err sorry I guess maybe `creatordate` instead of `committerdate`

Reply:
Switched to creatordate. Also tried `taggerdate` but it did not give the desired result.